base/docs/helpdb.jl

```diff
@@ -8064,30 +8061,28 @@ Join an array of `strings` into a single string, inserting the given delimiter b
 join
 
 doc"""
-```rst
-..  linreg(x, y) -> [a; b]
+    linreg(x, y) -> a, b
 
-Linear Regression. Returns ``a`` and ``b`` such that ``a+b*x`` is the closest line to the given points ``(x,y)``. In other words, this function determines parameters ``[a, b]`` that minimize the squared error between ``y`` and ``a+b*x``.
-
-**Example**::
-
-   using PyPlot;
-   x = float([1:12])
-   y = [5.5; 6.3; 7.6; 8.8; 10.9; 11.79; 13.48; 15.02; 17.77; 20.81; 22.0; 22.99]
-   a, b = linreg(x,y) # Linear regression
-   plot(x, y, "o") # Plot (x,y) points
-   plot(x, [a+b*i for i in x]) # Plot the line determined by the linear regression
+Perform linear regression. Returns `a` and `b` such that `a + b*x` is the closest
+straight line to the given points `(x, y)`, i.e., such that the squared error
+between `y` and `a + b*x` is minimized.
 
+**Example**:
+```
```

Comment:
No, actually, I don't -- the output in the ReST already magically says `code-julia`.